HarrisMoviesApplication/ViewController.swift

```diff
@@ -2,19 +2,42 @@
 //  ViewController.swift
 //  HarrisMoviesApplication
 //
-//  Created by juan harrington on 20/03/22.
+//  Created by Juan Harrington ✅ on 20/03/22.
```

Comment:
update this in [Xcode settings](https://stackoverflow.com/questions/8650661/how-to-change-developer-name-company-name-in-xcode):

Reply:
This is missing